src/Type/CompoundType.php

```diff
@@ -16,6 +17,6 @@ public function isAcceptedWithReasonBy(Type $acceptingType, bool $strictTypes):
 
 	public function isGreaterThan(Type $otherType): TrinaryLogic;
```

Comment:
What about here?

Reply:
Indeed, `var_dump('' < 0);` is true on PHP 8 but false on PHP 7.4